requirements.txt

```diff
@@ -0,0 +1,25 @@
+attrs==21.2.0
```

Comment:
Curious if all these dependencies are needed. The README states you only need `pytest`, `pytest-watch`, and `black`, but maybe all of these are required?